great_expectations/profile/basic_dataset_profiler.py

```diff
@@ -84,20 +86,24 @@ def _get_column_cardinality(cls, df, column):
                 cardinality = "many"
         # print('col: {0:s}, num_unique: {1:s}, pct_unique: {2:s}, card: {3:s}'.format(column, str(num_unique), str(pct_unique), cardinality))
 
+        df.set_config_value('interactive_evaluation', False)
+
         return cardinality
 
     @classmethod
     def _profile(cls, dataset):
-
-
         df = dataset
 
         df.set_default_expectation_argument("catch_exceptions", True)
 
         df.expect_table_row_count_to_be_between(min_value=0, max_value=None)
         df.expect_table_columns_to_match_ordered_list(None)
+        df.set_config_value('interactive_evaluation', False)
 
-        for column in df.get_table_columns():
+        columns = df.get_table_columns()
+        number_of_columns = len(columns)
+        for i, column in enumerate(columns):
```

Comment:
Adding column-level info here really increases the verbosity of the overall output, but I do like having the names of the columns present.

Outside the CLI module, we route all such messages through the logger. So two changes requested here: change the print statement to a log output, and ensure you don't use f-string style formatting to maintain py2 compatibility.

Reply:
It definitely increases the verbosity which is IMO far batter than indeterminate silence. Eventually I'd like to see a progress bar so this is a small step in that direction.

Reply:
Fixed both recommendations!